front/lib/api/workspace_statistics.ts

```diff
@@ -44,39 +42,19 @@ export async function computeWorkspaceStatistics(
     includeDeleted: true,
   });
 
-  const results = await concurrentExecutor(
-    dataSources,
-    async (dataSource) => computeDataSourceStatistics(dataSource),
-    { concurrency: DATA_SOURCE_STATISTICS_CONCURRENCY }
-  );
+  const results = await computeDataSourceStatistics(dataSources);
 
-  const hasError = results.some((r) => r.isErr());
-  if (hasError && !ignoreErrors) {
+  if (results.isErr()) {
     return new Err(
       new Error("Error computing statistics.", {
-        cause: removeNulls(
-          results.map((r) => (r.isErr() ? r.error.message : null))
-        ),
+        cause: results.error.message,
       })
     );
   }
 
-  const stats = results.reduce<WorkspaceStats>(
-    (acc, r) => {
-      if (r.isErr()) {
-        return {
-          ...acc,
-          notFoundDataSources: [
-            ...(acc.notFoundDataSources ?? []),
-            {
-              name: r.error.code,
-              error: r.error.message,
-            },
-          ],
-        };
-      }
```

Comment:
Is this edge-case common enough to grant for a dedicated error handling? cc @aubin-tchoi